src/late/npm-on-ipfs/index.js

```diff
@@ -0,0 +1,54 @@
+import { dialog } from 'electron'
+import i18n from 'i18next'
+import which from 'which'
+import { store } from '../../utils'
+import { createToggler } from '../utils'
+import { install, update, uninstall } from './lib'
+
+export default function (ctx) {
+  run()
+
+  const activate = (value, oldValue) => {
+    if (value === oldValue) return
+    if (value === true) return install()
+    return uninstall()
+  }
+
+  createToggler(ctx, 'npmOnIpfs', activate)
```

Comment:
NOTE: This has a bug. Any change will trigger this even if do it bellow. **CHECK**